### ISSUE TITLE
[REQUEST] :help 42 missing

### ISSUE THREAD

**Issue by glezmen** [mawww/kakoune#4628]
### Feature

**:help 42** command doesn't work, as in Vim

**Comment by Riteo** [comment]
Is this a vim easter egg? Why should kakoune implement such a feature? Please elaborate.

**Comment by Delapouite** [comment]
Yes, each text-editor is free to implement its own lore. Like the info's ASCII cat in the case of Kakoune for example.
I think this issue can be closed.
